core/src/modules/DynamicsModule/include/FreeDriftDynamics.hpp

```diff
@@ -0,0 +1,91 @@
+/*!
+ * @file FreeDriftDynamics.hpp
```

Comment:
How is adding this new class related to cleaning up and avoiding the copying of the arrays? Can you add more information to the PR?

Reply:
This PR needs to be merged after PR #522.

Reply:
This PR is now independent of #522